QRCoder.Xaml/Properties/AssemblyInfo.cs

```diff
@@ -1,35 +0,0 @@
-using System.Reflection;
-using System.Runtime.InteropServices;
-
-// Allgemeine Informationen über eine Assembly werden über die folgenden 
-// Attribute gesteuert. Ändern Sie diese Attributwerte, um die Informationen zu ändern,
-// die mit einer Assembly verknüpft sind.
-[assembly: AssemblyTitle("QRCoder")]
-[assembly: AssemblyDescription("QRCoder is a simple library, written in C#.NET, which enables you to create QR Codes. It's licensed under the MIT-license.")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("www.code-bude.net")]
-[assembly: AssemblyProduct("QRCoder")]
-[assembly: AssemblyCopyright("Copyright © www.code-bude.net / Raffael Herrmann.  All rights reserved.")]
-[assembly: AssemblyTrademark("written by Raffael Herrmann")]
-[assembly: AssemblyCulture("")]
```

Comment:
I'm concentrating on what appears in NuGet, not what's compiled into the assembly.  So some of these have been omitted in Directory.Build.props because NuGet ignores them.